.circleci/config.yml

```diff
@@ -91,7 +87,7 @@ jobs:
       - run: cd email_login/ && npm rebuild node-sass
       - run: 
           name: Login Build
-          command: cd email_login/ &&  yarn build
+          command: cd email_login/ && echo 'SKIP_PREFLIGHT_CHECK=true' >> .env && yarn build
```

Comment:
It's better to just set `SKIP_PREFLIGHT_CHECK=true` in the CircleCI environment variables config